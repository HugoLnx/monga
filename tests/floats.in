0.45315 3.98e5
-9.4f
927.356e3f
927.356e+3f
-927.3e-3f
+1.2e+3f
.9
9.
.9e+2f
9.e+2f
