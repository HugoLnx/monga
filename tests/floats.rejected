9.984ehikhnsaeh
927.356rty
927.356es
927.356ee
927.356e-
927.356e+
927.356e.
927.356e3e
927.356e3+
927.356e+3+
927.356ef
927.356ef3
927.356e-f
.e
.f
.e3
+.
.+
