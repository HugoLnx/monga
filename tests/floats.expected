FLOAT,0.45315
FLOAT,398000.00000
FLOAT,-9.40000
FLOAT,927356.00000
FLOAT,927356.00000
FLOAT,-0.92730
FLOAT,1200.00000
FLOAT,0.90000
FLOAT,9.00000
FLOAT,90.00000
FLOAT,900.00000
